common/ui_common.c

```diff
@@ -921,14 +919,16 @@ int adjust_wallpaper_element(lv_group_t *ui_group, int starter_image) {
                 lv_obj_move_foreground(ui_pnlBox);
                 break;
             case 3: // Fullscreen + Behind
+                lv_obj_set_y(ui_pnlBox, 0);
                 lv_obj_set_height(ui_pnlBox, device.MUX.HEIGHT);
-                lv_obj_set_align(ui_imgBox, LV_ALIGN_BOTTOM_RIGHT);
+                lv_obj_set_align(ui_imgBox, LV_ALIGN_CENTER);
```

Comment:
recommending to update this setting to be defaulted to center for both full screen modes to work better with higher resolution devices like RGCubeXX